lib/teams/emr-on-eks-team.ts

```diff
@@ -47,9 +47,13 @@ export interface EmrEksTeamProps extends TeamProps {
   */
   virtualClusterName: string,
   /**
-   * List of execution role to associated with the VC namespace
+   * List of execution role to associated with the VC namespace {@link ExecutionRoleDefinition}
    */
-  executionRoles: ExecutionRoleDefinition[] 
+  executionRoles: ExecutionRoleDefinition[],
```

Comment:
As I mentioned in my prev comment: It should be a semicolon `;` as opposed to comma and applied consistently across all interface members.

Reply:
fixed